configuration/structures.go

```diff
@@ -118,3 +118,46 @@ type Group struct {
 	AgentPriorities map[string]GroupPriority `json:"agent_priorities"`
 	RoutingStatus   string                   `json:"routing_status"`
 }
+
+// Agent defines basic Agent information
+type Agent struct {
+	ID                 string        `json:"id"`
+	Name               string        `json:"name,omitempty"`
+	Role               string        `json:"role,omitempty"`
+	AvatarPath         string        `json:"avatar_path,omitempty"`
+	JobTitle           string        `json:"job_title,omitempty"`
+	Mobile             string        `json:"mobile,omitempty"`
+	MaxChatsCount      uint          `json:"max_chats_count,omitempty"`
+	AwaitingApproval   bool          `json:"awaiting_approval,omitempty"`
+	Groups             []GroupConfig `json:"groups,omitempty"`
+	WorkScheduler      WorkScheduler `json:"work_scheduler,omitempty"`
+	Notifications      []string      `json:"notifications,omitempty"`
+	EmailSubscriptions []string      `json:"email_subscriptions,omitempty"`
+}
```

Comment:
I'm wondering if you should add support to all other fields not listed in docs 🤔

Reply:
if it's not listed in the docs, then it's not in the code :D SDK is public, docs are public, they should match each other

or do we have some undocumented public fields?

Reply:
We have a few of them, but I think that this most used and available to users are listed in docs.

Reply:
non-documented fields should not be listed here